win32ss/user/winsrv/usersrv/init.c

```diff
@@ -121,8 +121,32 @@ CreateSystemThreads(PVOID pParam)
 
 CSR_API(SrvCreateSystemThreads)
 {
-    DPRINT1("%s not yet implemented\n", __FUNCTION__);
-    return STATUS_NOT_IMPLEMENTED;
+    HANDLE ServerThread;
+    CLIENT_ID ClientId;
+    NTSTATUS Status;
+
+    Status = RtlCreateUserThread(NtCurrentProcess(),
```

Comment:
For threads within CSRSS, please use `CsrExecServerThread(thread_entry_point, 0)`.